docs/requirements_notebooks.txt

```diff
@@ -6,4 +6,5 @@ seaborn
 scipy
 scikit-learn
 polars==1.1.0
-pyarrow
\ No newline at end of file
+pyarrow
+hvplot
```

Comment:
I had prepared a PR that gets rid of a separate requirements file for notebooks:
- #1880

That doesn't need to move forward, but if it did, we'd probably want to be more conservative about adding new dependencies.